content/uk/docs/concepts/overview/components.md

```diff
@@ -0,0 +1,211 @@
+  * Job controller: Спостерігає за Job обʼєктами, що відповідають за одноразові задачі, а потім створє Pods, що будуть виконувати ці завдання до повного їх завершення.
+  * Endpoints controller: Заповнює Endpoint обʼєкти, тобто обєднує Services та Pods
+  * Service Account & Token controllers: Створює стандартний аккаунт та API token доступу
+
+<!--
+{{< glossary_definition term_id="cloud-controller-manager" length="short" >}}
+
+The cloud-controller-manager only runs controllers that are specific to your cloud provider.
+If you are running Kubernetes on your own premises, or in a learning environment inside your
+own PC, the cluster does not have a cloud controller manager.
+
+As with the kube-controller-manager, the cloud-controller-manager combines several logically
+independent control loops into a single binary that you run as a single process. You can
+scale horizontally (run more than one copy) to improve performance or to help tolerate failures.
+
+The following controllers can have cloud provider dependencies:
+
+  * Node controller: For checking the cloud provider to determine if a node has been deleted in the cloud after it stops responding
+  * Route controller: For setting up routes in the underlying cloud infrastructure
+  * Service controller: For creating, updating and deleting cloud provider load balancers
+-->
+
+### cloud-controller-manager
+
+{{< glossary_definition term_id="cloud-controller-manager" length="short" >}}
+
+The cloud-controller-manager only runs controllers that are specific to your cloud provider.
+If you are running Kubernetes on your own premises, or in a learning environment inside your
+own PC, the cluster does not have a cloud controller manager.
+
+As with the kube-controller-manager, the cloud-controller-manager combines several logically
+independent control loops into a single binary that you run as a single process. You can
+scale horizontally (run more than one copy) to improve performance or to help tolerate failures.
+
+The following controllers can have cloud provider dependencies:
+
+  * Node controller: For checking the cloud provider to determine if a node has been deleted in the cloud after it stops responding
+  * Route controller: For setting up routes in the underlying cloud infrastructure
+  * Service controller: For creating, updating and deleting cloud provider load balancers
+
+<!-- 
+Node components run on every node, maintaining running pods and providing the Kubernetes runtime environment.
+-->
+
+## Node компоненти
+
+Node компоненти запускаються на кожній node, підтримуючи запуск pods та забезпечуючи Kubernetes оточення
+
+### kubelet
+
+{{< glossary_definition term_id="kubelet" length="all" >}}
+
+### kube-proxy
+
+{{< glossary_definition term_id="kube-proxy" length="all" >}}
+
+### Container runtime
+
+{{< glossary_definition term_id="container-runtime" length="all" >}}
```

Comment:
Netlify build failed because `content/uk/docs/reference/glossary/container-runtime.md` is missing.

https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/6301338d0e8cae0008c00678#L86
> ERROR 2022/08/20 19:19:47 [uk] "docs/concepts/overview/components.md": "container-runtime" is not a valid glossary term_id, see ./docs/reference/glossary/* for a full list